fe/fe-core/src/main/java/com/starrocks/load/InsertOverwriteJobState.java

```diff
@@ -0,0 +1,11 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.load;
+
+public enum InsertOverwriteJobState {
+    OVERWRITE_PENDING,
+    OVERWRITE_PREPARED,
```

Comment:
Why not have running state? what's difference between pending and prepared?

Reply:
OVERWRITE_PREPARE is the same to OVERWRITE_RUNNING. I will change OVERWRITE_PREPARE to OVERWRITE_RUNNING.
OVERWRITE_PENDING is to get new partition names which are used to gc.